containers/containers.go

```diff
@@ -91,6 +91,10 @@ type Store interface {
 	// List returns containers that match one or more of the provided filters.
 	List(ctx context.Context, filters ...string) ([]Container, error)
 
+	// Stream returns containers that match one or more of the provided filters as a
```

Comment:
Should this be `ListStream` to match service in case we add new streaming methods?